src/js/player.js

```diff
@@ -4290,12 +4291,24 @@ class Player extends Component {
    * @return {string}
    *         The current language code when getting
    */
-  language(code) {
+  language(code, skipListeners) {
     if (code === undefined) {
       return this.language_;
     }
 
-    this.language_ = String(code).toLowerCase();
+    if (this.language_ !== String(code).toLowerCase()) {
+      this.language_ = String(code).toLowerCase();
+      // skipListeners helps to avoid trigger on initialization  when component is not evented
+      if (!skipListeners) {
+        /**
+        * fires when the player language change
+        *
+        * @fires Player#languagechange
+        * @type {EventTarget~Event}
+        */
+        this.trigger('languagechange');
```

Comment:
Would it make more sense to check if the component is evented or has a trigger function at this point rather than having a `skipListeners` argument?

Reply:
yeah, that makes sense. Didn't even think of that. We have `skipListeners` elsewhere, which is why I brought it up but `isEvented` definitely makes sense here.

Reply:
PR for it https://github.com/marcodeltorob/video.js/pull/2